Ruby/template/default_config/browser.yml

```diff
@@ -1,6 +1,6 @@
 &defaults_browser
 
-source: local # local, browserstack, perfecto, sauce_labs, kobiton, local_device
+source: local               # local, browserstack, perfecto, sauce_labs, kobiton, local_device
```

Comment:
you didn't add lambda_test to the commented examples, if that's something we still want to have

Reply:
done